drop-in/src/main/java/com/adyen/checkout/dropin/ui/DropInActivity.kt

```diff
@@ -585,6 +565,22 @@ class DropInActivity :
         }
     }
 
+    private fun handleWeChatPayResponse(intent: Intent) {
+        val actionFragment = getActionFragment() ?: return
```

Comment:
In what instances can `getActionFragment()` return null? Just returning would result in nothing happening. Would it be better to throw an error or show some kind of error dialog?

Reply:
It would happen if the user went back to the app and dismissed drop-in then went back to the browser and completed the payment, something very annoying indeed and we have an open issue about but I'm not sure how to deal with it now.